source/about/mattermost-mobile-releases.md

```diff
@@ -21,13 +21,13 @@ We strongly recommend using the latest mobile app release available that contain
-| v2.25 {ref}`FEATURE <release-v2-25-1>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.25.1) \| {ref}`Changelog <release-v2-25-1>` | 2025-02-16 | 2025-03-15 | {ref}`v10.5 <release-v10.5-extended-support-release>`, {ref}`v10.4 <release-v10.4-feature-release>`, {ref}`v10.3 <release-v10.3-feature-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>` |
-| v2.24 {ref}`FEATURE <release-v2-24-1>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.24.1) \| {ref}`Changelog <release-v2-24-1>` | 2025-01-16 | 2025-02-15 | {ref}`v10.4 <release-v10.4-feature-release>`, {ref}`v10.3 <release-v10.3-feature-release>`, {ref}`v10.2 <release-v10.2-feature-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>` |
-| v2.23 {ref}`FEATURE <release-v2-23-1>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.23.1) \| {ref}`Changelog <release-v2-23-1>` | 2024-12-16 | 2025-01-15 | {ref}`v10.3 <release-v10.3-feature-release>`, {ref}`v10.2 <release-v10.2-feature-release>`, {ref}`v10.1 <release-v10.1-feature-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>` |
-| v2.22 {ref}`FEATURE <release-v2-22-0>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.22.0) \| {ref}`Changelog <release-v2-22-0>` | 2024-11-15 | 2024-12-15 | {ref}`v10.2 <release-v10.2-feature-release>`, {ref}`v10.1 <release-v10.1-feature-release>`, {ref}`v10.0 <release-v10.0-major-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>`, {ref}`v9.5 <release-v9-5-extended-support-release>` |
-| v2.21 {ref}`FEATURE <release-v2-21-0>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.21.0) \| {ref}`Changelog <release-v2-21-0>` | 2024-10-16 | 2024-11-15 | {ref}`v10.1 <release-v10.1-feature-release>`, {ref}`v10.0 <release-v10.0-major-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>`, {ref}`v9.10 <release-v9-10-feature-release>`, {ref}`v9.5 <release-v9-5-extended-support-release>` |
-| v2.20 {ref}`FEATURE <release-v2-20-1>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.20.1) \| {ref}`Changelog <release-v2-20-1>` | 2024-09-16 | 2024-10-15 | {ref}`v10.0 <release-v10.0-major-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>`, {ref}`v9.10 <release-v9-10-feature-release>`, {ref}`v9.9 <release-v9-9-feature-release>`, {ref}`v9.5 <release-v9-5-extended-support-release>` |
+| v2.26 {ref}`FEATURE <release-v2-26-1>` [Download](https://github.com/mattermost/mattermost-mobile/releases/tag/v2.26.1) \| {ref}`Changelog <release-v2-26-1>` | 2025-03-16 | 2025-04-15 | {ref}`v10.6 <release-v10-6-feature-release>`, {ref}`v10.5 <release-v10-5-extended-support-release>`, {ref}`v10.4 <release-v10-4-feature-release>`, {ref}`v9.11 <release-v9-11-extended-support-release>` |
```

Comment:
Each of the changelog entries will also need to match:
![Screenshot 2025-03-21 at 8 37 25 AM](https://github.com/user-attachments/assets/7843ec0d-295a-4b6f-89b5-a74149f00b65)

Reply:
We can use periods or dashes -- whichever is preferred.

Reply:
@cwarnermm I'll fix them in a separate PR in order to unblock this PR.